mesh_to_grid_map_converter:
  numPointX: 200
  numPointY: 200
  waviness: 200
  xMin: -10
  yMin: -10
  zMin: -0.5
  xMax: 10
  yMax: 10
  zMax: 0.5
  seed: 5